test/python/expected/upgrade_validation/expected_dependency.out

```diff
@@ -506,12 +507,14 @@ Function sys.int2fixeddecimaldiv(smallint,sys.smallmoney)
 Function sys.int2rowversion(smallint,integer,boolean)
 Function sys.int2varbinary(smallint,integer,boolean)
 Function sys.int4_fixeddecimal_cmp(integer,sys.fixeddecimal)
+Function sys.int4_numeric_cmp(integer,numeric)
```

Comment:
We need a int4 to numeric = condition view test case to avoid add this dependency

Reply:
added